src/actions/search.js

```diff
@@ -1,21 +1,33 @@
 import actionTypes from '../constants/actions';
 import { loadingStarted, loadingFinished } from '../actions/loading';
 import { getAccount } from '../utils/api/account';
-import { getTransactions } from '../utils/api/transactions';
 import { getDelegate, getVoters, getAllVotes } from '../utils/api/delegate';
+import { getTransactions } from '../utils/api/transactions';
+import { getBlocks } from '../utils/api/blocks';
 import searchAll from '../utils/api/search';
+import transactionTypes from '../constants/transactionTypes';
 import { updateWallet } from './wallets';
 
 const searchDelegate = ({ publicKey, address }) =>
   (dispatch, getState) => {
     const liskAPIClient = getState().peers.liskAPIClient;
     getDelegate(liskAPIClient, { publicKey }).then((response) => {
-      dispatch({
-        data: {
-          delegate: response.data[0],
-          address,
-        },
-        type: actionTypes.searchDelegate,
+      getTransactions({
+        liskAPIClient, address, limit: 1, type: transactionTypes.registerDelegate,
+      }).then((transactions) => {
```

Comment:
same here, I know this is old code, but know you make changes is good to have it with async/await